src/query/block/types.go

```diff
@@ -119,8 +128,8 @@ type StepIter interface {
 type UnconsolidatedStepIter interface {
 	Iterator
 	StepMetaIter
-	// Current returns the current step for the block
-	Current() (UnconsolidatedStep, error)
+	// CurrentUnconsolidated returns the current step for the block
+	CurrentUnconsolidated() (UnconsolidatedStep, error)
```

Comment:
To have a single block that would satisfy both Consolidated and Unconsolidated block interfaces, otherwise it would have conflicts. Originally I had a single SeriesIterator and StepIterator that satisfied both interfaces, so had to make the change here, too, but since split it up, so can revert this